lang/cs/Org.Apache.REEF.Examples.HelloCLRBridge/ClrBridgeClient.cs

```diff
@@ -87,7 +87,7 @@ private static IConfiguration GetRuntimeConfiguration(string name)
             switch (name)
             {
                 case Local:
-                    var dir = Path.Combine(Directory.GetCurrentDirectory(), "REEF_LOCAL_RUNTIME");
+                    var dir = Path.Combine(".", "REEF_LOCAL_RUNTIME");
```

Comment:
In test scenario, test folder generated by the test framework is very long and have spaces in between, that makes command fail when calling java in a command line.